src/ConnectorEdgeModule/moduleSettings.example.json

```diff
@@ -0,0 +1,82 @@
+﻿{
+  "RootDirectory": null,
+  "KeyVaultBaseUrl": null,
+  "DisableSubscriptionRestoreService": true,
+  "EnableMessageFilter": false,
+  "ConnectorId": "###",
+  "Logging": {
+    "LogLevel": {
+      "Default": "debug",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "OpcUa": {
+    "DefaultServerBrowseDepth": 3,
+    "NodeBrowseDepth": 1,
+    "EnableRegisteredNodes": false,
+    "SubscriptionBatchSize": 100,
+    "ReadBatchSize": 100,
+    "RegisterNodeBatchSize": 100,
+    "AwaitSessionLockTimeoutSecs": 3,
+    "ReconnectIntervalSecs": 10,
+    "NativeSettings": {
+      "Topic": "configuration",
+      "ConsumerGroup": "...",
+      "ReadBatchSize": 9999
+    }
+  },
+  "Communication": {
+    "SchemaUrl": "https://someSchemaStore.com/schemas/",
+    "Shared": {
+      "Type": "kafka",
+      "NativeSettings": {
+        "BootstrapServers": "...",
+        "Username": "000",
+        "Debug": "consumer,cgrp,topic,fetch",
+        "Key": "....",
+        "saslMechanism": "PLAIN",
+        "saslUsername": "...",
+        "saslPassword": "...",
+        "securityProtocol": "SaslSsl",
+        "sslCaLocation": "ehCert/cacert.pem",
+        "socketKeepaliveEnable": "True",
+        "SocketTimeoutMs": 30000,
+        "acks": -1,
+        "TopicMetadataRefreshIntervalMs": 300000,
+        "autoOffsetReset": "earliest",
+        "enableAutoCommit": "False",
+        "enablePartitionEof": "False",
+        "statisticsIntervalMs": 5000,
+        "groupId": "..."
+      }
+    },
+    "TelemetryEndpoint": {
+      "Type": "kafka",
+      "UploadTimeoutMs": null,
+      "NativeSettings": {
+        "MaxRetries": 5, 
+        "WaitTimeBetweenRetriesInSeconds": 3, 
+        "Topic": "TelemetryTopic"
+      }
+    },
+    "CommandEndpoint": {
+      "Type": "kafka",
+      "NativeSettings": {
+        "Topic": "ComandTopic"
+      }
+    },
+    "ResponseEndpoint": {
+      "Type": "kafka",
+      "NativeSettings": {
+        "Topic": "ResponseTopic"
+      }
+    }
+  },
+  "Persistance": {
```

Comment:
Persistence is maybe too generic. Persistence of what? Logs, app data, messages?
We know it is for configuration, but from a user perspective, they may get the idea that they can persist other stuff?
Maybe inside persistence one can define instances of storage locations for different types of data?
I don't want to overthink this, just noting this down.

Reply:
What would be your suggestion. We also did not want to put a 'Configuration' section inside the 'Configuration' file. It is also to your point, configuration of what?
So I agree we could make it more obvious but I am not sure how?

Reply:
I think let's leave this for a later stage. The details and specifics of this can be worked out in another dedicated task.
We can evolve and refine this in future sprints.
This comment was more to raise awareness and get the thinking process going.